src/executeCommandLine/executeCommandLine.ts

```diff
@@ -741,6 +742,10 @@ export function executeCommandLine(
     cb: ExecuteCommandLineCallbacks,
     commandLineArgs: readonly string[],
 ) {
+    if (!commandLineArgs.find(arg => /^--?skipJSDocParsing$/.test(arg))) {
+        commandLineArgs = [...commandLineArgs, "--skipJSDocParsing"];
+    }

```

Comment:
I'm rethinking this bit; I feel like I should probably change the thing that actually parses the options (set this as a default), but it changes a lot more baselines too.